pkg/pillar/types/domainmgrtypes.go

```diff
@@ -51,6 +55,29 @@ func (config DomainConfig) GetOCIConfigDir() string {
 	}
 }
 
+// GetTaskName assigns a unique name to the task representing this domain
+// FIXME: given config.UUIDandVersion.Version part not sure config.AppNum is needed for uniqueness
+func (config DomainConfig) GetTaskName() string {
+	return config.UUIDandVersion.UUID.String() + "." +
+		config.UUIDandVersion.Version + "." +
+		strconv.Itoa(config.AppNum)
+}
+
+// DomainnameToUUID does the reverse of GetTaskName
+func DomainnameToUUID(name string) (uuid.UUID, error) {
+	// FIXME: we can likely drop this altogether
```

Comment:
We used to report the metrics for dom0 (the first line in xentop) using the zero uuid. I think that is still the case which means we can not drop this.